site/layouts/alias.html

```diff
@@ -0,0 +1,12 @@
+{{- if (strings.Contains .Permalink "v1.0.2") -}}
```

Comment:
didn't find a good way to make `v1.0.2` read from some place.

Reply:
cc @Xunzhuo